### PR TITLE
LGA-2753  replace old ga with tagman

### PR DESCRIPTION
**Pull request by digitalali-moj**
## What does this pull request do?

Remove old Google analytics code and add new Google Tag Manager code.  This makes Google analytics tracking work for all pages.

## Any other changes that would benefit highlighting?

Yes, this is very unlikely to ever change so is hard coded as it's simpler that way.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by said-moj — COMMENTED** [review]
Should this tag work and be the same across all environments(including local)?

**Comment by digitalali-moj** [comment]
> Should this tag work and be the same across all environments(including local)?

Yes this is for all environments and should work on all too. We can filter results in GA by domain when needed.

File: cla_frontend/templates/base.html [hunk 1]
```diff
@@ -24,6 +22,8 @@
     <!-- print styles -->
     <link href="{% staticmin 'stylesheets/cla.print.css' %}" media="print" rel="stylesheet" type="text/css" />
 
+    <script src="{% static 'javascripts/vendor/tagmanager.js' %}"></script>
```
Comment (psweeting1) [inline cla_frontend/templates/base.html:25]: Just a question, why in the head? 

Does tag manager need to load first on the page script wise or can it be called later to render just like the old script import was doing?

Comment (digitalali-moj) [reply cla_frontend/templates/base.html:25]: Great question. Google recommends to place it in the Head - how important that is would be debatable probably - but would need experimenting.

**Review by psweeting1 — APPROVED** [review]
LGTM

**Review by TawneeOwl — APPROVED** [review]
LGTM

**Pull request merged**